test/jointjs/paper.js

```diff
@@ -22,6 +22,51 @@ module('paper', {
     }
 });
 
+test('paper.addCell() number of sortViews()', function() {
+
+    var callback = sinon.spy();
```

Comment:
Well instead of triggering a new event (`sorted`) just for the purpose of tests I would rather use `sinon` in the following way.
```
var spy = sinon.spy(this.paper, 'sortViews');
// later
assert.equal(spy.callCount, 1);
```